lib/parsers/govendor-parser.ts

```diff
@@ -0,0 +1,52 @@
+import {GoPackageConfig} from '../types';
+import {InvalidUserInputError} from '../errors';
+import {VendorJson} from './types';
+
+function parseGovendorJsonContents(jsonStr: string): GoPackageConfig {
+    try {
+        const gvJson = JSON.parse(jsonStr) as VendorJson;
+
+        const goProjectConfig: GoPackageConfig = {
+            ignoredPkgs: [] as string[],
+            lockedVersions: {},
+            packageName: gvJson.rootPath!,
+        };
+
+        const packages = (gvJson.package || gvJson.Package);
+        if (packages) {
+            packages.forEach((pkg) => {
+                const revision = pkg.revision || pkg.Revision || pkg.version || pkg.Version;
+
+                const version = pkg.versionExact || ('#' + revision);
```

Comment:
template literals line 20

Reply:
This file was merely moved, but w/o a logical change.
IMO once I'll enforce a prettier this will be fixed